packages/webapp/src/components/Finances/Transaction/ExpandedContent/LabourTable.jsx

```diff
@@ -0,0 +1,182 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import React, { useState, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import styles from './styles.module.scss';
+import { useTranslation } from 'react-i18next';
+import EnhancedTable from '../../../Table/v2';
+import StateTab from '../../../RouterTab/StateTab';
+import { useCurrencySymbol } from '../../../../containers/hooks/useCurrencySymbol';
+
+export default function LabourTable({ transaction }) {
+  const { t } = useTranslation();
+  const currencySymbol = useCurrencySymbol();
+
+  const [activeTab, setActiveTab] = useState('employee');
+
+  const employeeTotalHours = useMemo(
+    () => calculateTotalHours(transaction.items.EMPLOYEE),
+    [transaction],
+  );
+
+  const taskTotalHours = useMemo(
+    () => calculateTotalHours(transaction.items.TASK_TYPE),
+    [transaction],
+  );
+
+  const employeeColumns = useMemo(
+    () => [
+      {
+        id: 'employee',
+        label: t('SALE.LABOUR.TABLE.EMPLOYEE'),
+        Footer: <div className={styles.footer}>{t('SALE.LABOUR.TABLE.DAILY_TOTAL')}</div>,
+        columnProps: {
+          style: { width: '33.33%' },
```

Comment:
@SayakaOno thank you for the 1/3rd width link! I couldn't figure out how to supply a fallback with a style object (specifically ESLint complained about duplicate keys)... do you know if it's possible? Otherwise I'll just keep the low-tech approach.

Reply:
I don't know if it's possible to set a fallback, sorry... but I believe calc is fine too!!